tables/vpc_flow_log/vpc_flow_log_table.go

```diff
@@ -1,138 +1,365 @@
 package vpc_flow_log
 
 import (
-	"log/slog"
-	"time"
-
-	"github.com/rs/xid"
-
+	"github.com/turbot/go-kit/helpers"
 	"github.com/turbot/pipe-fittings/v2/utils"
 	"github.com/turbot/tailpipe-plugin-aws/sources/cloudwatch_log_group"
 	"github.com/turbot/tailpipe-plugin-aws/sources/s3_bucket"
 	"github.com/turbot/tailpipe-plugin-sdk/artifact_source"
 	"github.com/turbot/tailpipe-plugin-sdk/artifact_source_config"
+	"github.com/turbot/tailpipe-plugin-sdk/constants"
+	"github.com/turbot/tailpipe-plugin-sdk/error_types"
 	"github.com/turbot/tailpipe-plugin-sdk/formats"
 	"github.com/turbot/tailpipe-plugin-sdk/row_source"
 	"github.com/turbot/tailpipe-plugin-sdk/schema"
 	"github.com/turbot/tailpipe-plugin-sdk/table"
+	"github.com/turbot/tailpipe-plugin-sdk/types"
 )
 
 const VpcFlowLogTableIdentifier = "aws_vpc_flow_log"
-
-const vpcFlowLogFormat = `$version $account-id $interface-id $srcaddr $dstaddr $srcport $dstport $protocol $packets $bytes $start $end $action $log-status`
+const VpcFlowLogTableNilValue = "-"
+const VpcFlowLogTableSkippedData = "SKIPDATA"
+const VpcFlowLogTableNoData = "NODATA"
```

Comment:
These are valid log entries, I don't think we should forcefully remove them in code but rather make a suggestion that users can filter them out using a `filter`?

Reply:
Hmm, good point, @cbruno10 - could we consider keeping the `SKIPDATA`/`NODATA` values as-is instead of skipping them? That said, this behavior would then apply to all columns. We still need to retain the check for `tp_timestamp` and `tp_data`, since those are required columns in the Tailpipe DB.

Reply:
Yes, I think I'd keep them, as users can filter out these values when collecting or in their queries like @graza-io mentioned. May be worth adding an example in the Example Configurations section showing how to avoid collecting these with `filter`.